### ISSUE TITLE
Add examples for each model in torchvision

### ISSUE THREAD

**Issue by fengyuentau** [pytorch/vision#7044]
### 🚀 The feature

Provide examples on how to do the correct and minimal pre- and post-processings in order to get the correct result. "Minimal" stands for the case when wrong results are produced if one of the pre- or post-processing is not performed.

### Motivation, pitch

Hi, thanks for your hard work to provide such a convenient way for researchers and developers to try out different models. However, I found it hard to find examples on how to do the correct pre- and post-processings in order to get the correct result. This will not be a problem for experienced researchers, but such examples can really help beginners.

Could you consider adding examples for different models in this repository?

**Comment by pmeier** [comment]
> Provide examples on how to do the correct and minimal pre- and post-processings in order to get the correct result.

Could you clarify what you mean by "correct"? Are you using a model with builtin weights and want to know what we used for a data pipeline as well as evaluation logic?

If so, we link the recipe we used in the documentation. For example, `model = models.alexnet()` will give you [these default weights](https://pytorch.org/vision/stable/models/generated/torchvision.models.alexnet.html#torchvision.models.AlexNet_Weights). The description links to [this recipe](https://github.com/pytorch/vision/tree/main/references/classification#alexnet-and-vgg).

**Comment by fengyuentau** [comment]
> Are you using a model with builtin weights and want to know what we used for a data pipeline as well as evaluation logic?

Yes, exactly. Thank you for the link. It really helps.

I found it not easy to discover this useful directory from the home page of this repo. Could you consider adding a section describing this in README so other people who need it can also find it more easily.

**Comment by fengyuentau** [comment]
Sure, I did. I visited the doc page, clicked into the [model and pre-trained weights](https://pytorch.org/vision/stable/models.html) in the Package Reference section, but ended up with:

```python
weights = ResNet50_Weights.DEFAULT
preprocess = weights.transforms()
```

I mean I could always look into the code to find out what `weights.transforms()` does, but what about just providing a clear and simple link to tell people the recipe to save some time?

**Comment by pmeier** [comment]
> I mean I could always look into the code to find out what `weights.transforms()` does, but what about just providing a clear and simple link to tell people the recipe to save some time?

I'm still not sure what you want here. `weights.transforms()` can be different for each weight, so there is no such general link. We can only provide a link on the per-weight basis, and we are doing that.

For example, the documentation states the following things for [`ResNet50_Weights`](https://pytorch.org/vision/stable/models/generated/torchvision.models.resnet50.html#torchvision.models.ResNet50_Weights):

> `ResNet50_Weights.DEFAULT` is equivalent to `ResNet50_Weights.IMAGENET1K_V2`.

> The inference transforms are available at `ResNet50_Weights.IMAGENET1K_V1.transforms` and perform the following preprocessing operations: Accepts `PIL.Image`, batched `(B, C, H, W)` and single `(C, H, W)` image `torch.Tensor` objects. The images are resized to `resize_size=[256]` using `interpolation=InterpolationMode.BILINEAR`, followed by a central crop of `crop_size=[224]`. Finally the values are first rescaled to `[0.0, 1.0]` and then normalized using `mean=[0.485, 0.456, 0.406]` and `std=[0.229, 0.224, 0.225]`.

> The inference transforms are available at `ResNet50_Weights.IMAGENET1K_V2.transforms` and perform the following preprocessing operations: Accepts `PIL.Image`, batched `(B, C, H, W)` and single `(C, H, W)` image `torch.Tensor` objects. The images are resized to `resize_size=[232]` using `interpolation=InterpolationMode.BILINEAR`, followed by a central crop of `crop_size=[224]`. Finally the values are first rescaled to `[0.0, 1.0]` and then normalized using `mean=[0.485, 0.456, 0.406]` and `std=[0.229, 0.224, 0.225]`.

Note that the `resize_size` is different between V1 and V2: `[256] vs. [232]`.

Plus, as explained above, each weight links to the recipe. Since that is can also be different for each weight, we also can't have a universal link there.

**Comment by pmeier** [comment]
Closing due to inactivity. @fengyuentau feel free to re-open in case I didn't address everything.
